example-web/src/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+root.render(
+  <React.StrictMode>
+    <App />
```

Comment:
I'm having issues running the example app for web.
I launch it by following the README step by step, but all I see is the empty page.

Reply:
I'm able to display the Slider if I just simplify the `<App/>` to render the `SliderExample` only, but this rendered Slider disappears as soon as there's any interaction with it.

Reply:
Hmmm, that's quite interesting. Let me see if I can recreate that by wiping my project folder entirely and rebuilding.

Are you getting any console errors when you see the whitescreen?

Reply:
No errors at all, but I can see that the server works well. It renders whatever I code and throws the error if I intentionally break something.
Also, I rendered the very basic example of Slider, and it does render, but the Slider cannot be interacted with - it just won't move. Let me try it using a different web browser, though...

Reply:
Yeah, it might be a browser polyfill issue. I'm working on Brave (Chrome/Chromium) and it seems to work there. What browser are you on?

Reply:
Aah, I see it - it's just the package that I had already built in `dist/`, but it required full rebuild. Sorry for confusion!